### DISCUSSION TITLE
command not found: chunkc

### DISCUSSION THREAD

**Discussion by szafran98** [asmvik/yabai#1613] [Q&A] [answered]
I already spent at least 5 hours to made skhd work, but i stuck. I want to configure macbook m1 via yabai and skhd. Yabai is working, skhd too i suppose, but keymaps didn't work. I restarted skhd with verbose parameter and on keypress i'm getting error `zsh: command not found: chunkc`. I can't find any solution, i've reinstalled both applications many times. I was even reading source code, but there is no any quote about this binary. What can i do in this case?

**Comment by 24unix** [comment]
chunkwm was the predecessor of yabai.
What is in your skhdrc?

**Comment by szafran98** [comment]
I just copied this example config: https://github.com/koekeishiya/skhd/blob/master/examples/skhdrc

**Comment by 24unix** [comment]
That looks very outdated.

**Comment by azinsharaf** [comment, marked as answer]
Yeah it seems outdated. I have yabai instead of chunkc and with different syntax.

**Comment by szafran98** [comment]
OMG, i waste whole night for nothing. I've installed skhd as homebrew and at the end of installation there is a notification with prepared command to copy example initial config. I just wanted to check if everything works fine, but it wasn't so i didn't check someone else configuration. I should ask here at the beginning, thank you all.

@azinsharaf Can you in addition give me your config? I will be very gratefoul.

**Reply by azinsharaf** [reply]
my `skhdrc` is simple. Here it is:
```
# focus space
cmd - 1 : yabai -m space --focus 1 
cmd - 2 : yabai -m space --focus 2
cmd - 3 : yabai -m space --focus 3
cmd - 4 : yabai -m space --focus 4
cmd - 5 : yabai -m space --focus 5
cmd - 6 : yabai -m space --focus 6
cmd - 7 : yabai -m space --focus 7
cmd - 8 : yabai -m space --focus 8
cmd - 9 : yabai -m space --focus 9
cmd - 0 : yabai -m space --focus 10

cmd - x : yabai -m space --focus recent
cmd - 0x32 : yabai -m space --focus recent
cmd - z : yabai -m space --focus prev
cmd - c : yabai -m space --focus next

# Navigation
cmd - h : yabai -m window --focus west
cmd - j : yabai -m window --focus south
cmd - k : yabai -m window --focus north
cmd - l : yabai -m window --focus east

# Moving windows
shift + cmd - h : yabai -m window --swap west
shift + cmd - j : yabai -m window --swap south
shift + cmd - k : yabai -m window --swap north
shift + cmd - l : yabai -m window --swap east

# Make window native fullscreen
cmd - f : yabai -m window --toggle zoom-fullscreen

# rotate tree
cmd - r : yabai -m window first --swap next && yabai -m window first --swap last

# cmd - t : yabai -m window --toggle float;\
          # yabai -m window --grid 4:4:1:1:2:2

# switch to an specific application
cmd - t : yabai -m space --focus teams
cmd - o : yabai -m space --focus outlook
cmd - b : yabai -m space --focus brave
cmd - a : yabai -m space --focus alacritty
cmd - g : yabai -m space --focus gis
cmd - n : yabai -m space --focus note
cmd - w : yabai -m space --focus workspot
cmd - d : yabai -m space --focus discord
cmd - s : yabai -m space --focus spotify

cmd - return : Open -a "Brave Browser MacOS.app" -n --args --new-window
```

**Reply by szafran98** [reply]
thank you so much, now everything works fine
